.github/workflows/contributors.yml

```diff
@@ -5,31 +5,41 @@ on:
     - cron:  '0 0 * * *'
 
 jobs:
+  delete-old-branch:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    steps:
+      - 
+        name: Delete old contributors-update branch
+        uses: dawidd6/action-delete-branch@v3
+        with:
+          github_token: ${{secrets.GITHUB_TOKEN}}
+          branches: contributors-update
+    
   add-contributors:
     runs-on: ubuntu-latest
+    needs: delete-old-branch
     steps:
-    - uses: actions/checkout@v2
-    - name: Delete old contributors-update branch
-      uses: dawidd6/action-delete-branch@v3
-      with:
-        github_token: ${{secrets.GITHUB_TOKEN}}
-        branches: contributors-update
-    - name: Create contributors-update branch
-      uses: peterjgrainger/action-create-branch@v2.0.1
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-        branch: 'contributors-update'
-    - name: Auto-add contributors
-      uses: BobAnkh/add-contributors@v0.2.2
-      with:
-        BRANCH: 'contributors-update'
-        PULL_REQUEST: 'master'
-        CONTRIBUTOR: '## Contributors'
-        COLUMN_PER_ROW: '6'
-        ACCESS_TOKEN: ${{secrets.GITHUB_TOKEN}}
-        IMG_WIDTH: '100'
-        FONT_SIZE: '14'
-        PATH: '/CONTRIBUTORS.md'
-        COMMIT_MESSAGE: 'docs(CONTRIBUTORS): update contributors'
-        AVATAR_SHAPE: 'round'
+      - 
```

Comment:
Can we avoid this syntax and put the name/uses in there

Reply:
Any reason for that or open for discussion? 😄 

Especially with many steps involved, this is much more readable. See our "default-on-push" with this style [here](https://gist.github.com/casperklein/7f839f94bf0070836737b111f04b52f6)

What do you think?

Reply:
Sorry, I don't wanted to pass you over. I forgot that auto-merge was activated and the other two approved it pretty fast.

Reply:
I did not see it as well, sorry:)

Reply:
What's your opinion about the style? As you approved it, I think you are fine with it?
If there is con-sense, I can provide a further PR to adjust the other workflows as well.

Reply:
No worries :)
I Just don't Like how this looks. Feels wrong for me and i Always remove those Lines If i have them Open in an Editor during Work. Better Option in my opinion would be to add empty Lines between the Tasks for better visibilty. 

No Matter which way we Go, i +1 your approach to Streamline the Syntax.